### ISSUE TITLE
`test_model_code_scripts.sh` is referring to not existing files

### ISSUE THREAD

**Issue by knuser** [facebookresearch/ELI5#4]
**Bug description**
It seems that `test_model_code_scripts.sh` is referring to not existing files.

Here is script output:
```
$ ./test_model_code_scripts.sh 
Traceback (most recent call last):
  File "process_data_to_source_target.py", line 130, in <module>
    train_q, train_a, train_d = read_data(args.input + "/train_data.json")
  File "process_data_to_source_target.py", line 18, in read_data
    with open(json_input) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'testing_files/train_data.json'
{'rouge-1': {'f': 0.19999999833333337, 'p': 0.19999999999999998, 'r': 0.19999999999999998}, 'rouge-2': {'f': 0.08333333166666669, 'p': 0.08333333333333333, 'r': 0.08333333333333333}, 'rouge-l': {'f': 0.1999999999998333, 'p': 0.19999999999999998, 'r': 0.19999999999999998}}
Traceback (most recent call last):
  File "pos_tag.py", line 67, in <module>
    valid_target, valid_source = open_files(args.input, "valid", args.dataset_name)
  File "pos_tag.py", line 25, in open_files
    with open(input_location + "/" + dataset + "." + dataset_name + "_target") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'testing_files/valid.test_target'
Namespace(beam=5, bpe=None, buffer_size=1, cpu=False, criterion='cross_entropy', data='/home/lab/fairseq/data-bin/eli5_data', dataset_impl=None, diverse_beam_groups=-1, diverse_beam_strength=0.5, force_anneal=None, fp16=False, fp16_init_scale=128, fp16_scale_tolerance=0.0, fp16_scale_window=None, gen_subset='test', input='-', lazy_load=False, left_pad_source='True', left_pad_target='False', lenpen=1, log_format=None, log_interval=1000, lr_scheduler='fixed', lr_shrink=0.1, match_source_len=False, max_len_a=0.0, max_len_b=500, max_sentences=1, max_source_positions=1024, max_target_positions=1024, max_tokens=None, memory_efficient_fp16=False, min_len=250.0, min_loss_scale=0.0001, model_overrides='{}', momentum=0.99, nbest=1, no_beamable_mm=False, no_early_stop=False, no_progress_bar=False, no_repeat_ngram_size=3, num_shards=1, num_workers=1, optimizer='nag', path='multitask_checkpoint.pt', prefix_size=0, print_alignment=False, quiet=False, raw_text=False, remove_bpe='@@ ', replace_unk=None, required_batch_size_multiple=8, results_path=None, sacrebleu=False, sampling=False, sampling_topk=-1, sampling_topp=-1.0, score_reference=False, seed=1, shard_id=0, skip_invalid_size_inputs_valid_test=False, source_lang='multitask_source_bpe', target_lang='multitask_target_bpe', task='translation', tbmf_wrapper=False, temperature=1.0, tensorboard_logdir='', threshold_loss_scale=None, tokenizer=None, unkpen=0, unnormalized=False, upsample_primary=1, user_dir=None, warmup_updates=0, weight_decay=0.0)
Traceback (most recent call last):
  File "/home/conda/fairseq/interactive.py", line 195, in <module>
    cli_main()
  File "/home/conda/fairseq/interactive.py", line 191, in cli_main
    main(args)
  File "/home/conda/fairseq/interactive.py", line 77, in main
    task = tasks.setup_task(args)
  File "/home/conda/fairseq/fairseq/tasks/__init__.py", line 19, in setup_task
    return TASK_REGISTRY[args.task].setup_task(args, **kwargs)
  File "/home/conda/fairseq/fairseq/tasks/translation.py", line 160, in setup_task
    src_dict = cls.load_dictionary(os.path.join(paths[0], 'dict.{}.txt'.format(args.source_lang)))
  File "/home/conda/fairseq/fairseq/tasks/fairseq_task.py", line 36, in load_dictionary
    return Dictionary.load(filename)
  File "/home/conda/fairseq/fairseq/data/dictionary.py", line 187, in load
    d.add_from_file(f, ignore_utf_errors)
  File "/home/conda/fairseq/fairseq/data/dictionary.py", line 204, in add_from_file
    raise fnfe
  File "/home/conda/fairseq/fairseq/data/dictionary.py", line 198, in add_from_file
    with open(f, 'r', encoding='utf-8') as fd:
FileNotFoundError: [Errno 2] No such file or directory: '/home/lab/fairseq/data-bin/eli5_data/dict.multitask_source_bpe.txt'
```

Missing files are (not present in this repository):
`testing_files/train_data.json`
`testing_files/valid.test_target`
`[...]fairseq/data-bin/eli5_data/dict.multitask_source_bpe.txt`

Where can I get those files?

**Comment by huihuifan** [comment]
Thanks for letting us know! Unfortunately the train_data.json file you need to generate yourself using the data creation scripts, and the multitask_source_bpe data assumes you have created the dataset and binarized it using fairseq-py. I've added some comments to clarify this.
